### DISCUSSION TITLE
Issue: Combining Breezy two factor authentication and Lock screen

### DISCUSSION THREAD

**Discussion by melvinmihaylovnulabg** [MarJose123/filament-lockscreen#59] [Q&A]
When the two packages are used in the same time and the user has to verify the authentication with code input for Breezy and lock screen is activated and the password is required for the login, when the password is correctly typed in it just reloads the page and the password is required again ... it never redirects to the page of Breezy authentication.

**Comment by MarJose123** [comment]
Hi 👋 I will check this during my free time if the integration with Breezy Plugin can be done smoothly.

**Reply by melvinmihaylovnulabg** [reply]
Thanks!

**Reply by melvinmihaylovnulabg** [reply]
@MarJose123 Update?

**Reply by MarJose123** [reply]
I'm integrating it right now, hoping it can be merging this week.

**Reply by MarJose123** [reply]
Can you share your use cases for me to understand it well.

**Reply by MarJose123** [reply]
Hi @melvinmihaylovnulabg I'm not using Breezy Plugin in my Filament Apps.

can you please try if this works. update your `composer.json` to 
```
"marjose123/filament-lockscreen" : "dev-2.x-breezy-integration"
```
and update your `filament-lockscren` config by adding
```php

    /**
     *  ==============================================================
     *  Filament Breezy
     *  https://github.com/jeffgreco13/filament-breezy
     *  ================================================
     */
    'breezy' => [
        '2fa' => [
            'enabled' => true,
        ],
    ],
```
and run `composer update`
